src/Watchlists/WatchList.jsx

```diff
@@ -95,7 +95,7 @@ export default function Watchlist({ item }) {
       {isOpen && (
         <>
           {isEmptyWatchList
-            ? <View style={styles.editorText}><Text style={{ color: '#ccc' }}>This WatchList is Empty</Text></View>
+            ? <View style={styles.editorText}><Text style={{ color: '#ccc' }}>This watchList is empty</Text></View>
```

Comment:
Please avoid unrelated changes. This should be another pullrequest

Reply:
Okay, sorry!